manifests/cf-manifest/deployments/040-cf-properties.yml

```diff
@@ -279,16 +279,20 @@ properties:
         client_key: (( grab secrets.consul_agent_key ))
         require_ssl: false
       dropsonde_port: (( grab properties.metron_agent.dropsonde_incoming_port ))
-      require_ssl: false
+      require_ssl: true
 
     converger:
       bbs:
         api_location: bbs.service.cf.internal:8889
         ca_cert: (( grab secrets.consul_ca_cert ))
-        client_cert: (( grab secrets.consul_agent_cert ))
-        client_key: (( grab secrets.consul_agent_key ))
+        client_cert: (( grab secrets.bbs_client_cert  ))
+        client_key: (( grab secrets.bbs_client_key  ))
+        require_ssl: true
+      ca_cert: (( grab secrets.bbs_ca_cert ))
+      server_cert: (( grab secrets.bbs_server_cert ))
+      server_key: (( grab secrets.bbs_server_key ))
```

Comment:
I can confirm that these options are not used at all. I removed them from my manifest and deployed again, no changes where applied to the deployed vms.

I think we can remove them safely.